data/yolo-obj.cfg

```diff
@@ -0,0 +1,788 @@
+[net]
```

Comment:
@Toz3wm Well, as my program do automatic modification, I told myself it could be nice to show the final result. Do you believe it is mandatory to remove it ?